warehouse/query-core/src/main/java/datawave/query/metrics/ShardTableQueryMetricHandler.java

```diff
@@ -560,6 +560,44 @@ public QueryMetricsDetailListResponse query(String user, String queryId, Datawav
         return response;
     }
     
+    // This method below needs to be changed to only show subplans
+    public QueryMetricsDetailListResponse subplan(String user, String queryId, DatawavePrincipal datawavePrincipal) {
```

Comment:
Yes this method needs to be changed.  Firstly I would expect a different response object to be returned.